autoware_launch/rviz/autoware.rviz

```diff
@@ -739,7 +739,7 @@ Visualization Manager:
               Name: DynamicObjects
               Namespaces:
                 label: true
-                path: false
+                path: true
                 path confidence: true
                 position covariance: true
```

Comment:
Position covariance flag is false for prediction.
https://github.com/tier4/autoware_launcher/blob/cdc3e2de5882445ad8d468a1788c95ed508a71c0/autoware_launch/rviz/autoware.rviz#L633-L638

Reply:
Fixed at 0da8afb .